src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-container.component.html

```diff
@@ -12,126 +12,131 @@
         </div>
     </div>
 </nav>
-<jhi-assessment-layout
-    (navigateBack)="navigateBack()"
-    [isLoading]="loadingParticipation"
-    [saveBusy]="saveBusy"
-    [submitBusy]="submitBusy"
-    [cancelBusy]="cancelBusy"
-    [nextSubmissionBusy]="nextSubmissionBusy"
-    [isTeamMode]="!!exercise?.teamMode"
-    [isAssessor]="isAssessor"
-    [isTestRun]="isTestRun"
-    [exerciseDashboardLink]="exerciseDashboardLink"
-    [isIllegalSubmission]="!!submission && submission.type! === SUBMISSION_TYPE_ILLEGAL"
-    [canOverride]="canOverride"
-    [result]="manualResult"
-    [assessmentsAreValid]="assessmentsAreValid"
-    [complaint]="complaint"
-    [exercise]="exercise"
-    [submission]="submission"
-    [hasAssessmentDueDatePassed]="hasAssessmentDueDatePassed"
-    [correctionRound]="correctionRound"
-    [isProgrammingExercise]="true"
-    (save)="save()"
-    (submit)="submit()"
-    (cancel)="cancel()"
-    (nextSubmission)="nextSubmission()"
-    (updateAssessmentAfterComplaint)="onUpdateAssessmentAfterComplaint($event)"
->
-    <ng-container *ngIf="submission; then assessment; else noSubmission"></ng-container>
-</jhi-assessment-layout>
-
-<ng-template #assessment>
-    <ng-container
-        *jhiExtensionPoint="overrideCodeEditor; context: { title: editorTitle, toolbar: editorToolbar, feedback: editorBottom, exercise: exercise, participation: participation }"
+<div [id]="'exercise-' + exercise?.id">
+    <jhi-assessment-layout
+        (navigateBack)="navigateBack()"
```

Comment:
note: This element probably has to many `@inputs`

Reply:
Although it does not really show in the git diff, the only thing I did was wrap the existing html code with one `div` that has the `id` on it. So changing the elements with the `div` would be something for a followup PR. But thanks for the hints!